src/dashboard/src/components/api/views.py

```diff
@@ -157,23 +182,14 @@ def status(request, unit_uuid, unit_type):
     return helpers.json_response(response)
 
 
+@_api_endpoint(expected_methods=['GET'])
 def waiting_for_user_input(request):
```

Comment:
Yes. Though something might depend on this "get all waiting units" endpoint, no? Why not provide 3:
1. /api/unit/waiting
2. /api/ingest/waiting
3. /api/transfer/waiting